### PR TITLE
[NativeAOT-LLVM] Make ImportRuntimeIlcPackageTarget override conditional

### PR DESCRIPTION
**Pull request by maraf**

### CODE REVIEW

**Comment by maraf** [comment]
@dotnet/nativeaot-llvm

File: src/coreclr/nativeaot/BuildIntegration/Microsoft.DotNet.ILCompiler.SingleEntry.targets [hunk 1]
```diff
@@ -85,15 +85,10 @@
...
-  <Target Name="ImportRuntimeIlcPackageTarget" Condition="'$(BuildingFrameworkLibrary)' != 'true' and '$(_targetArchitecture)' == 'wasm'" DependsOnTargets="$(ImportRuntimeIlcPackageTargetDependsOn)" BeforeTargets="Publish">
-    <Error Condition="'@(PackageDefinitions)' == ''" Text="The PackageDefinitions ItemGroup is required for target ImportRuntimeIlcPackageTarget" />
-
-    <PropertyGroup>
-      <IlcHostPackagePath Condition="'%(PackageDefinitions.Name)' == '$(_hostPackageName)'">%(PackageDefinitions.ResolvedPath)</IlcHostPackagePath>
-      <RuntimePackagePath Condition="'%(PackageDefinitions.Name)' == '$(_targetPackageName)'">%(PackageDefinitions.ResolvedPath)</RuntimePackagePath>
-    </PropertyGroup>
-
-  </Target>
+  <PropertyGroup>
```
Comment (SingleAccretion) [inline src/coreclr/nativeaot/BuildIntegration/Microsoft.DotNet.ILCompiler.SingleEntry.targets:89]:
```suggestion
  <!-- TODO-LLVM: <new issue number>. Remove once we upgrade to the .NET 10 SDK. -->
  <PropertyGroup>
```
Once the upstream integration settles in, `OverrideImportRuntimeIlcPackageTarget` will always be `false`.

**Review by SingleAccretion — APPROVED** [review]
LGTM modulo one comment.

File: src/coreclr/nativeaot/BuildIntegration/Microsoft.DotNet.ILCompiler.SingleEntry.targets (hunk 1 above)
Comment (maraf) [reply src/coreclr/nativeaot/BuildIntegration/Microsoft.DotNet.ILCompiler.SingleEntry.targets:89]: We can drop it entirely if everyone/everything migrates from packages to SDK integration

**Pull request merged**